swagger-api/swagger-ui
# added deeplinking as prop

### Description




### Motivation and Context







### How Has This Been Tested?
1. Clone the swagger-api/swagger-ui repo and made the changes in the code.
2. Execute the `flavors/swagger-ui-react/release/run.sh` script.
3. The compiled library files we need are in the `flavors/swagger-ui-react/dist` folder. Verify the timestamps match when you ran the run.sh script.
4. Copy the files to another folder , ran a example with the generated code.
5. Tested deeplinking by putting the appropriate link in the browser.
6. Repo => https://github.com/revaret/swagger-ui-react



### Screenshots (if appropriate):



## Checklist



### My PR contains... 


### My changes...

### Documentation

### Automated tests
